src/containers/BooksForm.js

```diff
@@ -2,18 +2,9 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { createBook } from '../actions';
+import categories from '../data/categories';
 
 const BooksForm = props => {
```

Comment:
**[Required from the previous milestone]** It seems that the previous TSE forgot to mention that we need to reset the state after we store book data in the redux-store. Please make this change. It won't take long. [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/HandleSubmit.png)